<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainWindow</class>
 <widget class="QWidget" name="AdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="proceedButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid purple;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>110</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="userTypeCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>89</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
</string>
   </property>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Customer</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid red;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
